index.bs

```diff
@@ -2758,44 +2793,69 @@ Appeal by Advisory Committee Representatives</h3>
 
 	An [=Advisory Committee representative=] initiates an [=appeal=] by sending a request to the [=Team=],
 	and should also share this request with the [=Advisory Committee=].
-	The request should say “I appeal this Decision”
+	The request <em class=rfc2119>should</em> say “I appeal this Decision”
 	and identify the decision,
-	and may also include their rationale for appealing the decision.
+	and <em class=rfc2119>may</em> also include their rationale for appealing the decision.
 
 	Note: See [[DECISION-APPEAL inline]] for a recommendation
 	on how to communicate an appeal request to the [=Team=] and the [=AC=].
 
-	Within one week the [=Team=] <em class="rfc2119">must</em> announce the appeal process
+	Within one week the [=Team=] <em class="rfc2119">must</em> initiate the [=appeal=]
+	in the form of a [=Advisory Committee Override=]
+	on the proposal to overturn the decision.
+
+	The conclusion of the [=Advisory Committee Override=] is final:
```

Comment:
```suggestion
	The conclusion of the [=Advisory Committee Override=] is final;
```

Reply:
or
```suggestion
	The conclusion of the [=Advisory Committee Override=] is final, and
```

Reply:
Not sure what's wrong with the colon. The part after the colon explains what it means for the conclusion to be final.

Reply:
There's nothing *wrong* with the colon; it's just that I find colons often confuse readers. The meaning of the sentence shouldn't be changed by any of the constructions above.